clang/lib/Headers/hlsl/hlsl_intrinsics.h

```diff
@@ -18,14 +18,21 @@ namespace hlsl {
 
 #define _HLSL_BUILTIN_ALIAS(builtin)                                           \
   __attribute__((clang_builtin_alias(builtin)))
-#define _HLSL_AVAILABILITY(environment, version)                               \
-  __attribute__((availability(environment, introduced = version)))
+#define _HLSL_AVAILABILITY(platform, version)                                  \
+  __attribute__((availability(platform, introduced = version)))
+#define _HLSL_AVAILABILITY_STAGE(platform, version, stage)                     \
```

Comment:
should we be updating the below intrinsics to include shader stages?

Reply:
I believe all of the intrinsics implemented so far are available for all shader stages, so no update is needed.